sessions/js-createelement/toast-messages/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+		<title>Toast Messages</title>
+		<link rel="stylesheet" href="./css/styles.css" />
+		<script src="./js/index.js" defer></script>
+	</head>
+	<body>
+		<div class="buttons">
+			<button type="button" data-js="add-button">Add toast message</button>
+			<button type="button" data-js="clear-button">Clear all toast messages</button>
+		</div>
+
+		<ol class="toast-messages-container" data-js="toast-messages-container">
+			<li class="toast-message">I'm a toast message.</li>
```

Comment:
In the other exercises BEM is consistently used. Maybe it should be added here as well. Something like that?
```suggestion
		<ol class="toast-container" data-js="toast-messages-container">
			<li class="toast-container__message">I'm a toast message.</li>
```

Reply:
Yes, absolutely!